### PR TITLE
[incubator/kafka] Add security context to kafka statefulset

### PR DESCRIPTION
**Pull request by isindir**
Signed-off-by: Eriks Zelenka <isindir@users.sf.net>

#### What this PR does / why we need it:

* Adds security context via values to statefullset

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

#### Please review

- @benjigoldberg

### CODE REVIEW

File: incubator/kafka/templates/statefulset.yaml [hunk 1]
```diff
@@ -224,6 +224,10 @@ spec:
           name: {{ include "kafka.fullname" . }}-metrics
           {{- end }}
       {{- end }}
+      {{- if .Values.securityContext }}
+      securityContext:
+        {{- .Values.securityContext | toYaml | nindent 8 }}
```
Comment (davidkarlsen) [inline incubator/kafka/templates/statefulset.yaml]: have it left-justified and then indent - makes it easier and consistent with the rest of the manifest(s).

File: incubator/kafka/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Apache Kafka is publish-subscribe messaging rethought as a distributed
   commit log.
 name: kafka
-version: 0.14.6
+version: 0.14.7
```
Comment (davidkarlsen) [inline incubator/kafka/Chart.yaml]:
```suggestion
version: 0.15.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: incubator/kafka/Chart.yaml (hunk 2 above)
Comment (isindir) [reply incubator/kafka/Chart.yaml]: Updated as requested

File: incubator/kafka/templates/statefulset.yaml (hunk 1 above)
Comment (isindir) [reply incubator/kafka/templates/statefulset.yaml]: Updated as requested

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**